### PR TITLE
Confirmation Summary

### LINKED ISSUE #1032: p3 Review your report page
https://www.figma.com/file/gzgsQIGQbO1wXZQzEHXhoe/Sprint-3?node-id=1014%3A20810

### PR DESCRIPTION
**Pull request by amazingphilippe**
Fixes #1032

### CODE REVIEW

**Comment by amazingphilippe** [comment]
The splitting has been done. Some summary blocks are working, some need more work. 

Now we can create separate issues to finish them one by one.

File: f2/src/App.js [hunk 1]
```diff
@@ -11,69 +11,74 @@ import { ThemeProvider, Flex, Link, CSSReset } from '@chakra-ui/core'
...
-            </SkipLink>
-            <WarningBanner>
-              <Trans id="banner.warning" />
-            </WarningBanner>
-            <PhaseBanner phase={<Trans id="banner.phase" />}>
-              <Trans id="banner.phaseText" />
-            </PhaseBanner>
-            <TopBanner lang={i18n.locale} bg="black" />
-          </header>
+    <IntlProvider locale={i18n.locale}>
```
Comment (sastels) [inline f2/src/App.js]: Are all the changes in this file just indentation because you added this provider?

File: f2/package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "react-dom": "^16.12.0",
     "react-final-form": "^6.3.3",
     "react-ga": "^2.6.0",
+    "react-intl": "^3.11.0",
```
Comment (sastels) [inline f2/package.json]: Is this for language-based formatting?

File: f2/src/App.js (hunk 1 above)
Comment (amazingphilippe) [reply f2/src/App.js]: Imndentation does look wonky. I'll pretify it

File: f2/package.json (hunk 2 above)
Comment (amazingphilippe) [reply f2/package.json]: Yes. Is this what failed snyk?

**Pull request merged**